src/bundle/ControllerArgumentResolver/ContentTreeChildrenQueryArgumentResolver.php

```diff
@@ -35,18 +33,16 @@ public function __construct(
 
     public function supports(Request $request, ArgumentMetadata $argument): bool
     {
-        return Criterion::class === $argument->getType()
+        return 'array' === $argument->getType()
```

Comment:
This causes that any `array $filter` controller argument will be resolved by this. This is clearly not the intent, as it could cause existing controllers to suddenly try and resolve filters using these.

Therefore, no. It cannot automatically resolve `array $filter` argument.

Reply:
Fixed: https://github.com/ibexa/admin-ui/pull/1328/commits/1287de018fc2dc992d0f312b53d976bbfce8fa3c